<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>猫狗分类器</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="view_original"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="view_features"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="result">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="status">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>缺少模型</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_train">
    <property name="title">
     <string>训练</string>
    </property>
    <addaction name="action_train_new"/>
    <addaction name="action_train_old"/>
   </widget>
   <widget class="QMenu" name="menu_predict">
    <property name="title">
     <string>预测</string>
    </property>
    <addaction name="action_predict"/>
    <addaction name="action_measure"/>
   </widget>
   <addaction name="menu_train"/>
   <addaction name="menu_predict"/>
  </widget>
  <action name="action_train_new">
   <property name="text">
    <string>训练模型</string>
   </property>
   <property name="toolTip">
    <string>使用新的训练数据训练模型</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_train_old">
   <property name="text">
    <string>加载模型</string>
   </property>
   <property name="toolTip">
    <string>加载预训练的模型</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_predict">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="toolTip">
    <string>打开一张图片进行预测</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_measure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>模型评估</string>
   </property>
   <property name="toolTip">
    <string>选择测试数据评估模型</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
